Nimble Templates/App.xctemplate/TemplateInfo.plist

```diff
@@ -1,215 +1,391 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
-	<key>Kind</key>
-	<string>Xcode.Xcode3.ProjectTemplateUnitKind</string>
-	<key>Identifier</key>
-	<string>co.nimblehq.dt.unit.singleViewApplication</string>
-	<key>Ancestors</key>
-	<array>
-		<string>co.nimble.dt.unit.cocoaTouchApplicationBase</string>
-	</array>
-	<key>Concrete</key>
-	<true/>
-	<key>Description</key>
-	<string>This template provides a starting point for an application that uses a single view.</string>
-	<key>SortOrder</key>
-	<integer>1</integer>
-	<key>Options</key>
-	<array>
+	<dict>
+		<key>Kind</key>
+		<string>Xcode.Xcode3.ProjectTemplateUnitKind</string>
+		<key>Identifier</key>
+		<string>co.nimblehq.dt.unit.singleViewApplication</string>
+		<key>Ancestors</key>
+		<array>
+			<string>co.nimble.dt.unit.cocoaTouchApplicationBase</string>
+		</array>
+		<key>Concrete</key>
+		<true/>
+		<key>Description</key>
+		<string>This template provides a starting point for an application that uses a single view.</string>
+		<key>SortOrder</key>
+		<integer>1</integer>
+
+		<key>Project</key>
 		<dict>
-			<key>Default</key>
-			<string>Swift</string>
-			<key>Identifier</key>
-			<string>languageChoice</string>
-			<key>Units</key>
+			<key>Configurations</key>
 			<dict>
-				<key>Objective-C</key>
+				<key>Dev Staging</key>
```

Comment:
Please update the README.md if we need to close the project and open it again to see the change